src/gf.c

```diff
@@ -2445,7 +2445,7 @@ void jl_method_table_activate(jl_methtable_t *mt, jl_typemap_entry_t *newentry)
                         // found that this specialization dispatch got replaced by m
                         // call invalidate_backedges(mi, max_world, "jl_method_table_insert");
                         // but ignore invoke-type edges
-                        invalidated = _invalidate_dispatch_backedges(mi, type, m, d, n, replaced_dispatch, ambig, max_world, morespec);
+                        invalidated |= _invalidate_dispatch_backedges(mi, type, m, d, n, replaced_dispatch, ambig, max_world, morespec);
```

Comment:
I don't know that this is quite right. I can see that it should be set if `invalidated = 1` happens above in the `mt` case, but I don't know that I see why it should persist across the doubly nested for loop otherwise.

Reply:
Good catch. That fixes the one puzzling aspect of that log.